docs/readme.md

```diff
@@ -57,7 +57,7 @@ instance with an open debugging port.
 
 Lighthouse can run against a real mobile device. You can follow the [Remote Debugging on Android (Legacy Workflow)](https://developer.chrome.com/devtools/docs/remote-debugging-legacy) up through step 3.3, but the TL;DR is install & run adb, enable USB debugging, then port forward 9222 from the device to the machine with Lighthouse.
 
-You'll likely want to use the CLI flags `--disable-device-emulation --disable-cpu-throttling` and potentially `--disable-network-throttling`.
+You'll likely want to use the CLI flags `--disable-device-emulation --disable-cpu-throttling`, `--port=9222` and potentially `--disable-network-throttling`. If you want to test on a mobile device without dealing with those flags, you could use `--remote-device` instead, which enables these features by default.
```

Comment:
Let's promote this a little more :) "You'll likely want to use the `--remote-device` flag which disables CPU and network throttling, disables device emulation, and defaults the port to 9222"